components/ai_chat/resources/page/components/input_box/index.tsx

```diff
@@ -67,18 +67,20 @@ function InputBox(props: InputBoxProps) {
     }
   }
 
-  const handleFocus = (event: React.FormEvent<HTMLTextAreaElement>) => {
-    if (!props.context.isMobile) {
-      return
+  // We don't want to handle that event on desktop
+  let handleFocusMobile
+  if (props.context.isMobile) {
+    handleFocusMobile = (event: React.FormEvent<HTMLTextAreaElement>) => {
+      props.onFocusInputMobile()
```

Comment:
sorry, this is my fault - I was thinking something like

`onFocus={isMobile && props.onFocusInputMobile}`

Reply:
that `onFocus={props.context.isMobile && props.onFocusInputMobile}` gives an error 
```
Type 'false | (() => void)' is not assignable to type 'FocusEventHandler<HTMLTextAreaElement> | undefined'.
  Type 'boolean' is not assignable to type 'FocusEventHandler<HTMLTextAreaElement>'.ts(2322)
index.d.ts(2350, 9): The expected type comes from property 'onFocus' which is declared here on type 'DetailedHTMLProps<TextareaHTMLAttributes<HTMLTextAreaElement>, HTMLTextAreaElement>'
```

Reply:
should be something like that
`onFocus={() => props.context.isMobile ? props.onFocusInputMobile : null}`